etc/smokercli.yaml

```diff
@@ -0,0 +1,5 @@
+# Example configuration file for the smoker client
+# plugin path: Python module containing the host discovery plugins. It has to
+#              be valid python module path.
+
+plugin_path: smoker.client.plugins
```

Comment:
Probably, when there is default set of plugins shipped with smoker. Then user should have possibility to run some custom plugins together with the default ones.